### ISSUE TITLE
Email notifications generated for workitem updates do not include details on what was updated

### ISSUE THREAD

**Issue by ldimaggi** [openshiftio/openshift.io#4591]
##### Issue Overview
The Planner generates an email notification when a workitem is updated. No content related to the update made to the work item is included in the email. See attached for an example.

##### Expected Behaviour
A user would reasonably expect to see content in the email notification.

##### Current Behaviour
No content is included in the email notification.

##### Steps To Reproduce

1. Enable email notifications
2. Edit/update a workitem in the Planner.

##### Additional Information
See attached .pdf file.
[planner_updates.pdf](https://github.com/openshiftio/openshift.io/files/2626152/planner_updates.pdf)

**Comment by rohitkrai03** [comment]
@sahil143 @nimishamukherjee Does frontend generates email notifications for any updates to Work Item? This is the backend service https://github.com/fabric8-services/fabric8-notification which sends email notification and there are no calls to this service from frontend. 
AFAIK the WIT backend is responsible to send emails for any updates in Work Item. This one is not related to UI

**Comment by sahil143** [comment]
@rohitkrai03 No, Frontend doesn't generate email notifications. WIT is responsible for email notifications.

**Comment by joshuawilson** [comment]
If anyone working on the backend needs help with HTML or CSS they can ask, but that is correct, this is controlled by the backend.

**Comment by baijum** [comment]
This issue is marked as a "bug" as the system was providing the expected behavior before. So, when was the bug introduced, any approximate time? Also, please provide screenshots of how it looked when it was working before.

**Comment by baijum** [comment]
@sbose78 I can see the above-described behavior in the notification. But I need to get answers to the above questions that I asked.

**Comment by kwk** [comment]
In August I prepared the fabric8-wit backend to help fix this problem:

> The notification system can then pick up the revision ID from the custom field in a message and fetch information about the changes made to a work item in that revision to generate better emails with the actual changed content inside.

I think I have a PR or at least a branch on the notification repo that uses this revision ID to fetch information about the changes that happened in the update.

Btw. this issue was reported a while back I think but under a different title.

**Comment by kwk** [comment]
Btw. it might make sense to wait with this issue when I'm back from PTO/sick leave. Then we'll have to opportunity to address this on a completely different level using events instead of directly telling the notification service what to do. I'll do a cabal on this when I'm back.
